bokeh/plotting/helpers.py

```diff
@@ -491,7 +525,8 @@ def func(self, *args, **kwargs):
     kwlines = []
     kws = glyphclass.properties() - set(glyphclass._args)
     for kw in sorted(kws):
-        if kw == "session": continue # TODO (bev) improve or remove
+        if kw == "session":
+            continue  # TODO (bev) improve or remove
```

Comment:
I strongly suggest limiting unrelated changes, because you may cause your self a lot of work in merge conflict resolution, and may bias reviewers to not focus on the subject of a PR (I know we shouldn't, but we are just people, usually with strong opinions).